docs/content/annotations.md

```diff
@@ -40,6 +40,7 @@ organizations | list of strings | A list of organizations related to the annotat
 related_resources | list of URLs | A list of URLs pointing to related resources/documentation. Read more [here](#related-resources).
 schemas | list of object | A list of associations between value paths and schema definitions. Read more [here](#schemas).
 scope | string; one of `package`, `rule`, `document`, `subpackages` | The scope on which the `schemas` annotation is applied. Read more [here](./#scope).
+title | string | A human-redable name for the annotation target. Read more [here](#title).
```

Comment:
Thanks for fixing this! I think it should come first, though. Would you mind shuffling this up?

Reply:
I thought so as well, but I noticed that the index table was in order with the content in the body, so I kept the same order. while I'm touching it, do you think it's worth to reorder like so:

1. scope
1. title
1. description
1. authors
1. organizations
1. schemas
7. custom

or I could just keep the order and move title to the top like you suggested.

Reply:
Would you mind reordering? That sounds reasonable.